### PR TITLE
cloudflare-wrangler2 2.7.0

### PR DESCRIPTION
**Pull request by samford**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR updates `cloudflare-wrangler2` to the latest version, 2.7.0.

Besides that, this also updates the formula to use `wrangler` as the command name instead of `wrangler2`.  `wrangler2` was used in the initial PR to avoid having to add `conflicts_with` calls but the official command name is `wrangler` (as in the [commands section of the `README`](https://github.com/cloudflare/wrangler2#commands)), so using `wrangler2` as the command name is likely to trip up users.

`wrangler2` is intended as a replacement for the previous Rust version and the Rust version is in the process of being deprecated (it won't be supported in the future). All new development happens in the `wrangler2` project and Cloudflare's [Wrangler documentation](https://developers.cloudflare.com/workers/wrangler/install-and-update/) references it instead of the Rust version, so I don't imagine there will be many users (if any) who want to have both the old Rust and current JavaScript version installed side by side at this point (i.e., 7 months after the initial 2.0.0 release).

Edit: I tweaked the `bin.install_symlink`call to install both the `wrangler` and `wrangler2` symlinks, so any existing users who have adapted to calling `wrangler2` won't be caught out.

### CODE REVIEW

**Review by p-linnane — DISMISSED** [review]

**Comment by cho-m** [comment]
Probably want 2 separate PRs.
- One for update to 2.7.0
- One CI-syntax-only for adding conflicts with, which should have 2 commits.

**Comment by chenrui333** [comment]
yeah, I think we should do two separate commits.

File: Formula/cloudflare-wrangler.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ class CloudflareWrangler < Formula
 
   uses_from_macos "zlib"
 
+  conflicts_with "cloudflare-wrangler2", because: "both install `wrangler` binaries"
```
Comment (chenrui333) [inline Formula/cloudflare-wrangler.rb:24]: this would lose the old bottle, might work moving to a different PR

**Comment by chenrui333** [comment]
created #120280 as a followup on the conflicts_with thing

**Review by cho-m — APPROVED** [review]
